lib/models/payment.model.dart

```diff
@@ -32,12 +32,12 @@ enum PaymentType {
 
 extension PaymentTypeExtension on PaymentType {
   static const displays = {
-    PaymentType.receiveOnChain: "Bitcoin Received",
-    PaymentType.sendOnChain: "Bitcoin Sent",
+    PaymentType.receiveOnChain: "You received",
```

Comment:
I'm not sure whether "You" is needed (here and elsewhere). consider just "Received"

Reply:
I `you received` because it speaks to me :)